### DISCUSSION TITLE
Commercially available OTBRs?

### DISCUSSION THREAD

**Discussion by jseide** [openthread/openthread#9542] [Q&A] [answered]
Are there any non-matter commerciallly available OTBRs?  

It seems like apple/google/amazon OTBRs can only commission matter devices and lock you in to their “Home” ecosystem- is that right?

If I want to just grab the network credentials from an Apple/Amazon/Google OTBR and commission a thread device to them without using their Home app or matter, would that be possible?

**Comment by jwhui** [comment, marked as answer]
iOS and Android provide APIs for sharing Thread network credentials among mobile apps:

- https://developers.home.google.com/thread
- https://developer.apple.com/documentation/threadnetwork/
